Dev implement grafana auth interface (#1)
canonical/grafana-auth-interface: README.md

# grafana-auth-interface

## Description

This project contains libraries for the grafana-auth relationship. It contains both the provider and the requirer side of them.

> > Note: The charm located here is a placeholder charm and should not packed nor should be deployed.

Reviewer: ```suggestion
> Note: The charm located here is a placeholder charm and should not be packed nor  deployed.
```

Author: Done

> > Note: The charm located here is a placeholder charm and should not packed nor should be deployed.

Reviewer 2: ```suggestion
> Warning: The charm located here is a placeholder charm and should not packed nor should be deployed.
```

Author: Done

## Usage

This library can be used by any charm requiring or providing this interface. From the charm's
root directory, run:

```bash
charmcraft fetch-lib charms.grafana_auth_interface.v0.grafana_auth_interface
```

> 

Reviewer 2: There should also be something about the libs necessary for those charms to add to their `requirements.txt` file.

Reviewer 2: In this case, only `jsonschema`

Author: Done

# grafana-auth-interface

## Description

> This project contains libraries for the grafana-auth relationship. It contains both the provider and the requirer side of them.

Reviewer 3: ```suggestion
This project contains libraries for the grafana-auth relationship. It contains both the provider and the requirer sides.
```

Author: Fixed

> > Warning: The charm located here is a placeholder charm and should not be packed nor deployed.

Reviewer 3: Could / should it be used as the tester charm? ([example](https://github.com/canonical/observability-libs/blob/main/tests/integration/test_compute_resources_patch.py#L44))
